packages/sources/trueusd/src/endpoint/trueusd.ts

```diff
@@ -35,7 +27,7 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
-  const resultPath = validator.validated.data.resultPath
+  const resultPath = validator.validated.data.resultPath || 'totalTrust'
```

Comment:
We have something for this by adding:
```
export const endpointResultPaths = {
  trueusd: 'totalTrust'
}
```

It will fill in here https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/core/bootstrap/src/lib/external-adapter/builder.ts#L82